public/assets/css/main.css

```diff
@@ -0,0 +1,51 @@
+/* Cairo Google font */
```

Comment:
This an output file. Please, gitignore this file as it could be generated.

Reply:
it could be generated , but if gitignore it you cant keep track new changes

Reply:
No need for the new changes as this is an output file, and the real changes that we need to track is the Sass file.